packages/client/src/styles/globals.css

```
@@ -2357,8 +2514,36 @@ summary[class*="cursor-pointer"],
+    padding-left: max(0.5rem, env(safe-area-inset-left)) !important;
+    padding-right: max(0.5rem, env(safe-area-inset-right)) !important;
+  }
+
+  [data-chat-mode="roleplay"] .rpg-narrator-msg {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  [data-chat-mode="roleplay"] .mari-roleplay-message-row {
+    --mari-roleplay-mobile-avatar-space: calc((2.5rem * var(--roleplay-avatar-scale, 1)) + 0.5rem);
+    gap: 0.5rem;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)